meshtastic/interfaces/ble/compatibility_service.py

```
@@ -0,0 +1,193 @@
+
+if TYPE_CHECKING:
+    from meshtastic.interfaces.ble.interface import BLEInterface
+
+
+class BLECompatibilityEventService:
+    """Service helpers for compatibility event publication paths."""
+
+    @staticmethod
+    def _enqueue_publish_callback(
+        publishing_thread: object,
+        callback: Any,
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `callback`

(ANN401)